packages/react-meteor-data/useFind.ts

```diff
@@ -51,7 +52,7 @@ const useFindReducer = <T>(data: T[], action: useFindActions<T>): T[] => {
 // which has fields _mongo and _cursorDescription.
 const checkCursor = <T>(cursor: Mongo.Cursor<T> | Partial<{ _mongo: any, _cursorDescription: any }> | undefined | null) => {
   if (cursor !== null && cursor !== undefined && !(cursor instanceof Mongo.Cursor) &&
-      !(cursor._mongo && cursor._cursorDescription)) {
+    !(cursor._mongo && cursor._cursorDescription)) {
```

Comment:
Side note: is this the official Meteor formatting? I'm not against it or something, but it'd be best not to readjust it in every PR.

(Note that the newly added import uses `"` but the rest of the code uses `'`.)